src/phoneField/index.tsx

```diff
@@ -120,16 +122,19 @@ export default class PhoneField extends PureComponent<PhoneFieldProps, PhoneFiel
     countriesWhiteList: this.props.countriesWhiteList,
     [FIELDS.PHONENUMBER]: this.props.defaultValue,
     [FIELDS.PHONEPREFIX]: iso2toDialCode(this.props.defaultPrefix),
+    phoneRegion: [this.props.defaultPrefix],
     completePhoneNumber: iso2toDialCode(this.props.defaultPrefix) + this.props.defaultValue,
   }
 
   onChange = ({ name, value }: OnChangeParameters) => {
     this.setState({ [name]: value }, () => {
+      const iso2Codes = allCountriesCode[Number(this.state[FIELDS.PHONEPREFIX])]
       this.props.onChange({
         name: this.props.name,
         value: {
           [FIELDS.PHONENUMBER]: this.state[FIELDS.PHONENUMBER],
           [FIELDS.PHONEPREFIX]: this.state[FIELDS.PHONEPREFIX],
+          phoneRegion: !isEmpty(iso2Codes) ? iso2Codes : [''],
```

Comment:
Can not we use an empty array instead of this empty magic string?

Reply:
Done